components/brave_wallet/browser/unstoppable_domains_multichain_calls.h

```diff
@@ -79,10 +74,8 @@ class UnstoppableDomainsMultichainCalls {
 
  private:
   // domain -> call
-  base::flat_map<std::string, UnstoppableDomainsMultichainCall<ResultType>>
-      calls_;
+  base::flat_map<std::string, MultichainCall<ResultType>> calls_;
 };
-
-}  // namespace brave_wallet
+}  // namespace brave_wallet::unstoppable_domains
```

Comment:
nit: keep the newline

Reply:
https://github.com/brave/brave-core/pull/13202/commits/4f916563f379813df4f0438dbd77bad3f10553d2